assets/src/styles/campaigns/themes/_theme_antarctic.scss

```diff
@@ -1,3 +1,8 @@
+:root {
+  --link--color: #c44bc4;
```

Comment:
We could also move the variables to the top, and use this SCSS syntax to be able to use the variable in a `:root` selector, like `#{$antarctic-purple}`. But this way is fine for me too :+1: